content/docs/overview.md

```diff
@@ -204,6 +204,22 @@ my_variable: int // cannot be accessed outside this file.
 ```
 `@(private)` is equivalent to `@(private="package")`.
 
+### Authoring a package
+
+A package is a directory of Odin code files which are marked with the same package header: `package main`
+Each file in a directory _must_ have the same package declaration. A directory cannot contain multiple packages. 
+
+### Subpackages
```

Comment:
I would call this `### Organizing packages`. They're not technically sub-packages.

They're not properly sub-packages, even though it's easy to think of them as such.

Perhaps:
```
### Organizing packages

Packages may be thematically organized by placing them in subdirectories of another package. For example: `core:image/png` and `core:image/tga`, as subdirectories of `core:image`. 
Nesting these packages is a helpful taxonomy. It does not imply a dependency: `core:foo/bar` does not need to import `core:foo` and reference anything from it.


```

Reply:
Agreed, I think having the overview be clear, factual statements is the way to go. I'll update.